src/client/clu_client_setup.c

```diff
@@ -155,15 +167,6 @@ int wolfCLU_Client(int argc, char** argv)
         ret = _addClientArg(clientArgv, noClientCert, &clientArgc);
     }
 
-    if (ret == WOLFCLU_SUCCESS) {
-        /* add TLS downgrade support i.e -v d to arguments */
-        ret = _addClientArg(clientArgv, "-v", &clientArgc);
```

Comment:
downgrade should be kept as a default

Reply:
Restored. Looks like wolfssl_get_session will have to be tweaked for TLS 1.3